action.yml

```
@@ -47,7 +47,7 @@ runs:
 
         # Include executables in the `assets` folder - used for ServiceControl, ServicePulse, ServiceInsight, etc.
         if (Test-Path -Path assets) {
-          cp assets/* ./packaging/content
+          cp -Recurse assets/* ./packaging/content
```

Methinks that's illegal.
```suggestion
          cp -R assets/* ./packaging/content
```

---

As discussed on Slack, turns out that `cp` is an alias for the `Copy-Item` cmdlet on Windows, but not on other platforms. 🙄